### PR TITLE
PEP8 all the things.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by mozilla-travis** [comment]
```
snippets/settings/docker.py:27:1: E402 module level import not at top of file
snippets/settings/travis.py:27:1: E402 module level import not at top of file
snippets/urls.py:14:1: E402 module level import not at top of file
snippets/base/views.py:27:1: E731 do not assign a lambda expression, use a def

```

**Comment by mozilla-travis** [comment]
```
snippets/urls.py:14:1: E402 module level import not at top of file
snippets/settings/docker.py:27:1: E402 module level import not at top of file
snippets/settings/travis.py:27:1: E402 module level import not at top of file
snippets/base/views.py:27:1: E731 do not assign a lambda expression, use a def

```

**Comment by mozilla-travis** [comment]
```
local configuration: in /home/travis/build/mozilla/snippets-service
checking snippets/settings/docker.py
snippets/settings/docker.py:27:1: E402 module level import not at top of file
checking snippets/urls.py
snippets/urls.py:14:1: E402 module level import not at top of file
checking snippets/base/helpers.py
checking snippets/base/util.py
checking snippets/settings/travis.py
snippets/settings/travis.py:27:1: E402 module level import not at top of file
checking snippets/base/urls.py
checking snippets/base/tests/test_decorators.py
checking snippets/base/encoders.py
checking snippets/base/tests/test_fields.py
checking snippets/settings/base.py
checking snippets/base/tests/test_encoders.py
checking snippets/base/fields.py
checking snippets/base/managers.py
checking snippets/base/tests/test_middleware.py
checking snippets/base/__init__.py
checking snippets/base/tests/__init__.py
checking snippets/__init__.py
checking snippets/base/tests/test_util.py
checking snippets/base/storage.py
checking snippets/base/tests/test_forms.py
checking snippets/base/tests/test_cron.py
checking snippets/settings/test.py
checking snippets/base/cron.py
checking snippets/base/views.py
snippets/base/views.py:27:1: E731 do not assign a lambda expression, use a def
checking snippets/base/forms.py
checking snippets/base/tests/test_admin.py
checking snippets/base/tests/test_managers.py
checking snippets/base/admin.py
checking snippets/settings/__init__.py
checking snippets/base/models.py
checking snippets/base/middleware.py
checking snippets/base/tests/test_models.py
checking snippets/base/decorators.py
checking snippets/base/tests/test_views.py
directory snippets
directory snippets/settings
directory snippets/base
directory snippets/base/tests
directory snippets/base/templates
directory snippets/base/templates/base
directory snippets/base/templates/base/includes
directory snippets/base/static
directory snippets/base/static/templates
directory snippets/base/static/js
directory snippets/base/static/js/lib
directory snippets/base/static/img
directory snippets/base/static/html
directory snippets/base/static/css

```

**Comment by mozilla-travis** [comment]
```
local configuration: in /home/travis/build/mozilla/snippets-service
checking snippets/base/middleware.py
checking snippets/settings/travis.py
checking snippets/base/__init__.py
checking snippets/base/encoders.py
checking snippets/base/helpers.py
checking snippets/base/util.py
checking snippets/urls.py
checking snippets/base/urls.py
checking snippets/base/tests/test_decorators.py
checking snippets/settings/__init__.py
checking snippets/base/tests/test_fields.py
checking snippets/base/fields.py
checking snippets/settings/base.py
checking snippets/settings/test.py
checking snippets/__init__.py
checking snippets/base/tests/test_util.py
checking snippets/base/tests/test_encoders.py
checking snippets/base/tests/test_middleware.py
checking snippets/base/tests/__init__.py
checking snippets/base/managers.py
checking snippets/base/tests/test_forms.py
checking snippets/base/tests/test_cron.py
checking snippets/base/cron.py
checking snippets/settings/docker.py
checking snippets/base/decorators.py
checking snippets/base/forms.py
checking snippets/base/views.py
checking snippets/base/tests/test_admin.py
checking snippets/base/admin.py
checking snippets/base/storage.py
checking snippets/base/tests/test_managers.py
checking snippets/base/tests/test_models.py
checking snippets/base/models.py
checking snippets/base/tests/test_views.py
directory snippets
directory snippets/settings
directory snippets/base
directory snippets/base/static
directory snippets/base/static/css
directory snippets/base/static/templates
directory snippets/base/static/html
directory snippets/base/static/js
directory snippets/base/static/js/lib
directory snippets/base/static/img
directory snippets/base/templates
directory snippets/base/templates/base
directory snippets/base/templates/base/includes
directory snippets/base/tests

```

**Comment by glogiotatidis** [comment]
Now that @Osmose  is not around let's pep8 all the things!

**Comment by glogiotatidis** [comment]
@pmclanahan r?

File: snippets/base/views.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
 from snippets.base.util import get_object_or_none
 
 
-_http_max_age = lambda: getattr(settings, 'SNIPPET_HTTP_MAX_AGE', 90)
+def _http_max_age():
+    return getattr(settings, 'SNIPPET_HTTP_MAX_AGE', 90)
```
Comment (pmclanahan) [inline snippets/base/views.py]: Shouldn't there be 2 blank lines after a module level function definition?

**Comment by Osmose** [comment]
> Now that @Osmose is not around let's pep8 all the things!

YOU FOOLS BLINDLY FOLLOWING YOUR GOD PEP8 WILL ONLY LEAD TO PAIN AND SUFFERING

I CANNOT SAVE YOU IF YOU DO NOT WISH TO BE SAVED

**Comment by pmclanahan** [comment]
Looks good. I just have that one question about the function spacing. r+wc.

File: snippets/base/views.py (hunk 1 above)
Comment (glogiotatidis) [inline snippets/base/views.py]: quoting pep8  `Separate top-level function and class definitions with two blank lines.` It doesn't say anything about seperating top-level functions and variables, so I believe that's OK. 

https://www.python.org/dev/peps/pep-0008/#blank-lines

**Comment by glogiotatidis** [comment]
> YOU FOOLS BLINDLY FOLLOWING YOUR GOD PEP8 WILL ONLY LEAD TO PAIN AND SUFFERING
> 
> I CANNOT SAVE YOU IF YOU DO NOT WISH TO BE SAVED

My god ordered that I lint all your projects! Behold PEP8 is coming! :godmode:

**Comment by glogiotatidis** [comment]
thanks @pmclanahan !

**Pull request merged**